ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1320,6 +1320,11 @@ periodics:
   spec:
     containers:
     - args:
+      - --github-token-path=/etc/github/oauth
```

Comment:
Not this one, it's already at the end of the list (but it makes sense to move it here)

Reply:
Perhaps it's a nice opportunity to create a new token with more restricted permissions.

Reply:
agh, so stupid mistake, haven't noticed

Reply:
... but corrected now